extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -80,7 +81,8 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
   const qidoDicomWebClient = staticWado
     ? new StaticWadoClient(qidoConfig)
     : new api.DICOMwebClient(qidoConfig);
-  const wadoDicomWebClient = new api.DICOMwebClient(wadoConfig);
+
+  const wadoDicomWebClient = new StaticWadoClient(wadoConfig);
```

Comment:
ALireza requested with have the staticWado ? .... style instead of replacing the thing entirely